### ISSUE TITLE
Import profile crash

### ISSUE THREAD

**Issue by wizardx95** [meefik/linuxdeploy#503]
I had a backup file image and linux.xml
I restore them in my new device LG F160K.
After import profile "linux.xml" the app always crash when I hit properties button.

**Comment by mionec1** [comment]
Your profile is broken.
Do you see "Updating configuration file... done"?

**Comment by wizardx95** [comment]
Yes. I saw it. I fixed it by open profile .xml in a text viewer and set a new profile as old profile. And its work
